docs/source/specs/openapi.json

```diff
@@ -730,7 +730,7 @@
             "content": {
               "application/json": {
                 "schema": {
-                  "$ref": "#/components/schemas/RoleWithAccess"
+                  "$ref": "#/components/schemas/RolePagination"
```

Comment:
Hey, @astrozzc - for this, I think we _do_ actually get back `RoleWithAccess` when creating a new role, so I don't think this should change?
```
{
  "uuid": "b0dd3a46-7f9e-45ab-8af7-a23ae5777ee4",
  "name": "Test Role",
  "description": "Test Role",
  "access": [
    {
      "permission": "cost-management:*:*",
      "resourceDefinitions": []
    }
  ],
  "applications": [
    "cost-management"
  ],
  "system": false,
  "platform_default": false,
  "created": "2019-12-06T12:39:48.848177Z",
  "modified": "2019-12-06T12:39:48.848189Z"
}
```

Reply:
Ah, thanks for catching this, this is a mistake.